chain/standard_cheat_code_contract.go

```diff
@@ -125,23 +129,23 @@ func getStandardCheatCodeContract(tracer *cheatCodeTracer) (*CheatCodeContract,
 		func(tracer *cheatCodeTracer, inputs []any) ([]any, *cheatCodeRawReturnData) {
 			// Maintain our changes until the transaction exits.
 
-			// Obtain our spoofed difficulty
+			// // Obtain our spoofed difficulty
+			// TODO: Check chain config here to see if the EVM version is before 'Paris'.
+			// if t.evmContext.ChainConfig.LatestFork(t.evmContext.Time)
+			// // Change difficulty in block context.
+			// originalDifficulty := new(big.Int).Set(tracer.chain.pendingBlockContext.Difficulty.Big())
+			// tracer.chain.pendingBlockContext.Difficulty.Set(spoofedDifficulty)
+			// tracer.CurrentCallFrame().onTopFrameExitRestoreHooks.Push(func() {
+			// 	tracer.chain.pendingBlockContext.Difficulty.Set(originalDifficulty)
+			// })
```

Comment:
can we remove this? also, can you update the comment around the difficulty cheatcode's purpose?
Right now it says that it "Sets VM block number".